src/analysisd/logtest.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright (C) 2015-2020, Wazuh Inc.
+ * Copyright (C) 2009 Trend Micro Inc.
+ * All right reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation
+ */
+
+#include "shared.h"
+#include "rules.h"
+#include "decoders/decoder.h"
+#include "eventinfo.h"
+#include "../headers/pthreads_op.h"
+#include "../headers/defs.h"
+#include "../headers/validate_op.h"
+#include "../os_net/os_net.h"
+
+
+/**
+ * @brief A sessionLogtest instance represents a client.
+*/
+typedef struct sessionLogtest {
```

Comment:
Please add `w_` and `_t`, and keep the underscore notation. I think that "logtest session" would be more proper. On the other hand, you're calling all new functions `w_logtest_*`.
```c
typedef struct w_logtest_session_t { ...
```